site/content/en/docs/tasks/run/multikueue/kuberay.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Run Kuberay Jobs in Multi-Cluster"
+linkTitle: "Kuberay"
+weight: 4
+date: 2025-03-19
+description: >
+  Run a MultiKueue scheduled Kuberay Jobs.
+---
+
+## Before you begin
+
+Check the [MultiKueue installation guide](/docs/tasks/manage/setup_multikueue) on how to properly setup MultiKueue clusters.
+
+For the ease of setup and use we recommend using at least Kueue v0.11.0 and for Kuberay Operator at least v1.3.1.
+
+See [Kuberay Operator Installation](https://docs.ray.io/en/latest/cluster/kubernetes/getting-started/raycluster-quick-start.html#step-2-deploy-a-kuberay-operator) for installation and configuration details of Kuberay Operator.
+
+{{% alert title="Note" color="info" %}}
+Before the [ManagedBy feature](https://github.com/ray-project/kuberay/issues/2544) was supported in Kueue (below v0.11.0), the installation of Kuberay Operator in the <b>Manager Cluster</b> must be limited to CRDs only.
+
+To install the CRDs run:
+```bash
+kubectl create -k "github.com/ray-project/kuberay/ray-operator/config/crd?ref=v1.3.0"
+```
+{{% /alert %}}
+
+## MultiKueue integration
+
+Once the setup is complete you can test it by running a RayJob [`ray-job-sample.yaml`](/docs/tasks/run/rayjobs/#example-rayjob).
+
+{{% alert title="Note" color="primary" %}}
+Note: Kueue defaults the `spec.runPolicy.managedBy` field to `kueue.x-k8s.io/multikueue` on the management cluster for Kuberay Jobs (RayJob, RayCluster, RayService). 
```

Comment:
spec.managedBy?

Reply:
Thank you for catching this!
I opened the fixing PR: https://github.com/kubernetes-sigs/kueue/pull/4713